src/UserFrontend.php

```diff
@@ -67,6 +70,16 @@ public static function auth_cookie_expiration($duration, $user_id, $remember) {
     return $duration * 15;
   }
 
+  /**
+   * @implements wp_enqueue_scripts
+   */
+  public static function cookie_notice_enqueue_styles() {
+    $ver = Plugin::get_plugin_data()['Version'];
+    if (file_exists(Plugin::getBasePath() . '/dist/styles/cookie-notice.css')) {
```

Comment:
Why would this file not exist?  This condition can be removed I think.

Reply:
Removed.